src/ophys_etl/workflows/db/schemas.py

```diff
@@ -37,6 +37,46 @@ class WorkflowStepRun(SQLModel, table=True):
     end: datetime.datetime
 
 
+class MotionCorrectionRun(SQLModel, table=True):
+    """Motion correction run"""
+    __tablename__ = 'motion_correction_run'
+
+    workflow_step_run_id: int = Field(
+        foreign_key='workflow_step_run.id',
+        primary_key=True
+    )
+    max_correction_up: float
+    max_correction_down: float
+    max_correction_right: float
+    max_correction_left: float
+
+
+class OphysROI(SQLModel, table=True):
```

Comment:
Do you need to add some of the other outputs of the segmentatio(+postprocessing) pipeline like the exclusion flags (e.g. motion_boarder)?

Reply:
My thinking on the exclusion labels is that they are used for classifying the ROI so should be worked on when the classification step is worked on. I will make sure to incorporate the motion border when working on roi classification. Does this plan work with you?

Reply:
So the plan would be to access the "exclusion_label" from the json after the fact and put it in the classification step? The Prostprocess portion of the ophys_etl segmentation wrapper is calculating and producing these labels.

Reply:
I plan on just recalculating it when doing classification and not use the value calculated in this json. The other label this segment_posprocess calculates is "too_small", which I disagree with adding since the roi classifier can figure that out and don't plan on using.  The "motion_border" label can be an additional label in addition to the "Classifier_score>0.5" label I plan on adding which will be populated in the classification step.

Reply:
Sounds good.